docs/it/log_management.rst

```diff
@@ -0,0 +1,50 @@
+.. include:: ../common/common_definitions.rst
+
+.. _Log_Management:
+
+Retention Policy
+================
+
+Gestione dei Log di un OP e di un RP
+------------------------------------
+
+Gli OP e gli RP DEVONO mantenere 
+
+1. Un registro delle transazioni contenente i log relativi ai messaggi scambiati. I messaggi memorizzati e mantenuti nel registro DEVONO essere almeno i seguenti:
+
+    - **AuthenticationRequest**
+    - **AuthenticationResponse** relativa all'*AuthenticationRequest*
+    - **TokenRequest** relativa all'*AuthenticationRequest*
+    - **TokenResponse** relativa alla *TokenRequest* 
+    - L'eventuale **UserInfoRequest** relativa alla *TokenRequest*
+    - L'eventuale **UserInfoResponse** relativa alla *UserInfoRequest*
+    - L'eventuale **RevocationRequest** relativa alla *TokenRequest*
+    - L'eventuale **RevocationResponse** relativa alla *RevocationRequest*
+
+2. Un registro di federazione contenente, per ogni *AuthenticationRequest*, la **Trust Chain** relativa all'entità con la quale si stanno scambiando i messaggi. Nel caso di registro mantenuto da un OP che riceve una richiesta di autenticazione da un RP, DEVE contenere:
+
+    - L'**Entity Configuration** del RP richiedente.
+    - L'eventuale **Entity Configuration** del SA (nel caso di RP aggregato).
+    - L'eventuale **Entity Statement** del SA riferito al RP.
```

Comment:
definire eventuale, perché potrebbe essere interpretato come opzionale

devono conservare l'intera trust chain, composta come definito nello standard e con l'aggiunta obbligatoria dell'EC del TA (che in Federation è opzionale)

Reply:
ok